app.py

```
@@ -102,127 +106,101 @@ def listt():
 
 
 @app.route("/api/send/<int:target_user_id>")
+# api/send/{recipient}?gift_type={電電點|抽獎券}count={count}
 def send(target_user_id):
     if not session:
-        return jsonify({"resulet": "you must loggin", "status": 403})
+        return jsonify({"result": "you must login", "status": 403})
     try:
         api_admin = session.get(
             "user"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)